runtime/util/ConfigurationSerializer.cpp

```diff
@@ -327,6 +329,36 @@ void serializeConfigurationInternal(
   }
 }
 
+void serializeConfigurations(
+    const char *filename, std::unordered_set<block *, HashBlock, KEq> results) {
+  FILE *file = fopen(filename, "w");
+  auto state = serialization_state();
+
+  auto w = writer{file, nullptr};
+  auto size = results.size();
+  if (size == 0) {
+    emitConstantSort(state.instance, "SortGeneratedTopCell");
+    emitSymbol(state.instance, "\\bottom{}", size, 1);
+  } if (size == 1) {
+    auto result = *results.begin();
+    serializeConfigurationInternal(&w, result, nullptr, false, &state);
```

Comment:
```suggestion
    serializeConfigurationInternal(&w, results[0], nullptr, false, &state);
```

Reply:
I don't think that it's possible to get the element by index on a `std::unordered_set`.

Reply:
Of course - my bad!